src/controllers/user.controller.js

```diff
@@ -0,0 +1,78 @@
+const userService = require('../services/user.service.js');
+
+const get = (_req, res) => {
+  res.sendStatus(200);
+  res.send(userService.getAll());
+};
+
+const getById = (req, res) => {
+  const { id } = req.params;
+
+  const user = userService.getById(id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(user);
+};
+
+const create = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
```

Comment:
After sending a response with `res.sendStatus(400)`, you should return immediately to prevent further execution of the function.